__tests__/main.spec.ts

```diff
@@ -1094,42 +1094,89 @@ test('exempt pr labels will not be marked stale', async () => {
   expect(processor.staleIssues).toHaveLength(2); // PR should get processed even though it has an exempt **issue** label
 });
 
-test('exempt issue labels will not be marked stale and will remove the existing stale label', async () => {
-  expect.assertions(3);
-  const opts = {...DefaultProcessorOptions};
-  opts.exemptIssueLabels = 'Exempt';
-  const TestIssueList: Issue[] = [
-    generateIssue(
+describe('remove-stale-from-exempt-items is used', () => {
+  test('remove-stale-from-exempt-items is enabled, exempt issues that were recently updated will have stale removed', async () => {
+    expect.assertions(3);
+    const opts = {
+      ...DefaultProcessorOptions,
+      removeStaleWhenUpdated: true,
+      removeStaleFromExemptItems: true
+    };
+    opts.exemptIssueLabels = 'Exempt';
+    const TestIssueList: Issue[] = [
+      generateIssue(
+        opts,
+        1,
+        'My first issue',
+        '2020-01-01T17:00:00Z',
+        '2020-01-01T17:00:00Z',
+        false,
+        ['Exempt', 'Stale']
+      )
+    ];
+    const processor = new IssuesProcessorMock(
       opts,
-      1,
-      'My first issue',
-      '2020-01-01T17:00:00Z',
-      '2020-01-01T17:00:00Z',
-      false,
-      ['Exempt', 'Stale']
-    )
-  ];
-  const processor = new IssuesProcessorMock(
-    opts,
-    async p => (p === 1 ? TestIssueList : []),
-    async () => [
-      {
-        user: {
-          login: 'notme',
-          type: 'User'
-        },
-        body: 'Body'
-      }
-    ], // return a fake comment to indicate there was an update
-    async () => new Date().toDateString()
-  );
-
-  // process our fake issue list
-  await processor.processIssues(1);
-
-  expect(processor.staleIssues.length).toStrictEqual(0);
-  expect(processor.closedIssues.length).toStrictEqual(0);
-  expect(processor.removedLabelIssues.length).toStrictEqual(1);
+      async p => (p === 1 ? TestIssueList : []),
+      async () => [
+        {
+          user: {
+            login: 'notme',
+            type: 'User'
+          },
+          body: 'Body'
+        }
+      ], // return a fake comment to indicate there was an update
+      async () => new Date().toDateString()
+    );
+
+    // process our fake issue list
+    await processor.processIssues(1);
+
+    expect(processor.staleIssues.length).toStrictEqual(0);
+    expect(processor.closedIssues.length).toStrictEqual(0);
+    expect(processor.removedLabelIssues.length).toStrictEqual(1);
```

Comment:
Thanks for testing both the positive and negative cases! 👏 

I notice the assertions here are slightly different from the ones in the negative case: 

```js
    expect(processor.statistics?.processedIssuesCount).toStrictEqual(1);
    expect(processor.closedIssues).toHaveLength(0);
    expect(processor.removedLabelIssues).toHaveLength(0);
```

**Any reason not to test them the same way here?**

```suggestion
    expect(processor.statistics?.processedIssuesCount).toStrictEqual(1);
    expect(processor.staleIssues).toHaveLength(0);
    expect(processor.closedIssues).toHaveLength(0);
    expect(processor.removedLabelIssues).toHaveLength(1);
```

Reply:
Good catch! Totally agree